rustls/src/anchors.rs

```diff
@@ -87,8 +87,7 @@ impl RootCertStore {
     }
 
     /// Return the Subject Names for certificates in the container.
-    #[deprecated(since = "0.20.7", note = "Use OwnedTrustAnchor::subject() instead")]
-    pub fn subjects(&self) -> DistinguishedNames {
+    pub(crate) fn subjects(&self) -> DistinguishedNames {
```

Comment:
I'm confused. What's the rationale for undeprecating this?

Reply:
Yeah, I'm going to do that. 👍🏻